lib/matplotlib/backends/backend_ps.py

```diff
@@ -1452,6 +1452,10 @@ def convert_psfrags(tmpfile, psfrags, font_preamble, custom_preamble,
     # multiple
     if sys.platform == 'win32': precmd = '%s &&'% os.path.splitdrive(tmpdir)[0]
     else: precmd = ''
+    #Replace \\ for / so latex does not think there is a function call
+    latexfile = latexfile.replace("\\", "/")
```

Comment:
It will probably be also broken on unix, if you add a testcase where you set `TMP`, `TMPDIR` and `TEMP` (whatever mkstemp used first is probably enough) to a path with a `~` in it. Would probably enough to do that in the ps tests by simply adding a testcase which sets that in `os.environ` before doing the mpl calls...